taxcalc/functions.py

```diff
@@ -120,23 +179,82 @@ def ItemDed(_posagi, e17500, e18400, e18425, e18450, e18500, e18800, e18900,
                  ID_Charity_frt, puf):
 
     """
-    WARNING: Any additional keyword args, such as 'puf=True' here, must be 
+    Itemized Deduction; Form 1040, Schedule A
+    
+    Notes
+    -----
+    Tax Law Parameters:
```

Comment:
We should be consistent about whether we leave taxpayer characteristics or tax law parameters first. Do you have a preference? I am leaning towards taxpayer characteristics first as in Adjustments().